docs/run.md

```diff
@@ -171,7 +173,7 @@ Refer to [Microsoft tests](microsoft_tests.md) know more about how test tiers de
 If you want to validate more scenarios, run with `<tier id>` from 1 to 4. To run tier 1 to 4, it needs legacy LISAv2 tests. The LISA must be run in a Windows computer, and prepare a secret file. Learn more from [how to run legacy test cases](run_legacy.md).
 
 ```bash
-lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>" -v tier:<tier id>
+lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "private_key_file:<private key file>" -v tier:<tier id>
```

Comment:
It uses to keep consistent arguments between runs. What's reason to change the name?

Reply:
Using `admin_private_key_file`, its throwing error (saying private_key_file is missing) as the variable and the field key used in this [runbook](https://github.com/microsoft/lisa/blob/main/microsoft/runbook/ready.yml#L19) is `private_key_file` rather than `admin_private_key_file`.

Reply:
good catch! can you change the variable to `admin_private_key_file` in the runbook? It can keep consistent at user level, we will make it consistent later in schema level.

Reply:
Updated the runbook variable to `admin_private_key_file` and changed the parameter back to `admin_private_key_file` in `run.md`